docs/Configuration.md

```diff
@@ -1,5 +1,60 @@
 # Configuration
 
+Default configuration file is located in `config/` folder. All configuration options should be listed in the default configuration file `config/environment.default.js`. Please do not change this file directly! To change the default configuration values, create local files that override the parameters you need to change:
+
+-	Create a new `environment.dev.js` file in `config/` for `devel` environment;
+-	Create a new `environment.prod.js` file in `config/` for `production` environment;
+
+Some few configuration options can be overridden by setting environment variables. These and the variable names are listed below.
+
+## Nodejs server
+When you start dspace-angular on node, it spins up an http server on which it listens for incoming connections. You can define the ip address and port the server should bind itsself to, and if ssl should be enabled not. By default it listens on `localhost:3000`. If you want it to listen on all your network connections, configure it to bind itself to `0.0.0.0`.
+
+To change this configuration, change the options `ui.host`, `ui.port` and `ui.ssl` in the appropriate configuration file (see above):
+```
+module.exports = {
+  // Angular Universal server settings.
+  ui: {
+    ssl: false,
+    host: 'localhost',
+    port: 3000,
+    nameSpace: '/'
+  }
+};
+```
+
+Alternately you can set the following environment variables. If any of these are set, it will override all configuration files:
+```
+  DSPACE7_SSL=true
```

Comment:
Should we consider renaming these environment variables to simply `DSPACE_SSL` and similar?  Having a DSpace version number in the environment variable seems odd to me.  E.g. We don't want to have to rename these when DSpace 8 comes out.